pkgs/development/libraries/openldap/default.nix

```diff
@@ -20,6 +20,7 @@ stdenv.mkDerivation rec {
       "--disable-dependency-tracking"   # speeds up one-time build
       "--enable-modules"
       "--sysconfdir=/etc"
+      "--enable-crypt"
```

Comment:
Sounds like a reasonable choice. Can you retarget to this pull request to the `staging` branch?